packages/core-data/src/selectors.ts

```diff
@@ -951,11 +1100,13 @@ export function __experimentalGetCurrentThemeBaseGlobalStyles( state ) {
 /**
  * Return the ID of the current global styles object.
  *
- * @param {Object} state Data state.
+ * @param  state Data state.
  *
- * @return {string|null} The current global styles ID.
+ * @return The current global styles ID.
  */
-export function __experimentalGetCurrentThemeGlobalStylesVariations( state ) {
+export function __experimentalGetCurrentThemeGlobalStylesVariations(
+	state: State
+): string | null {
```

Comment:
ibid.

Reply:
What do you mean by ibid here? I know how it's used in bibliographies but I'm having a hard time translating that knowledge to this code review.

Reply:
Didn't mean to complicate things: it means "see the previous comment" or "same comment." Just didn't see the need to write the same comment multiple times.

Reply:
but I now see that were returning `string | null` so maybe I misread that and thought I had seen `any | null`